utils/micrond/files/etc/init.d/micrond

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+USE_PROCD=1
+
+CRONDIR=/usr/lib/micron.d
+
+service_triggers() {
+	for file in $CRONDIR/*; do
+		if [ -f /etc/config/${file##*/} ]; then
+			procd_add_reload_trigger "${file##*/}"
+		fi
+	done
```

Comment:
I don't think this reload trigger is useful. There is usually no relation between changes to UCI config and the need to reload crontabs. The reload_service function can be dropped as well.